check-diff.sh

```diff
@@ -548,7 +547,7 @@ function run_phpunit_local {
 			if [ -n "$PHPUNIT_CONFIG" ] || [ -e phpunit.xml* ]; then
 				phpunit $( if [ -n "$PHPUNIT_CONFIG" ]; then echo -c "$PHPUNIT_CONFIG"; fi )
 			fi
-			for nested_project in $( find $PATH_INCLUDES -name 'phpunit.xml*' -mindepth 2 ! -path '*/vendor/*' -name 'phpunit.xml*' -exec dirname {} \; ); do
+			for nested_project in $( find $PATH_INCLUDES -name 'phpunit.xml*' ! -path '*/vendor/*' -name 'phpunit.xml*' -exec dirname {} \; ); do
```

Comment:
@westonruter I just tested again. Setting mindepth on my side to 2 results in no tests running. The reason is that the $PATH_INCLUDES variable is set to:

wp-content/*/strathcom*

The phpunit.xml file resides in the root of a plugin directory, so the mindepth level would have to be 1 for this to work (and it does with 1 on my side). Does that make sense?